ZuluContent/Zulu/Mobiles/BaseCreature.Props.cs

```diff
@@ -101,13 +101,13 @@ public partial class BaseCreature : Mobile, IEnchanted, IZuluClassed, IElemental
         public virtual bool CanFly { get; set; } = false;
 
         [CommandProperty(AccessLevel.GameMaster)]
-        public virtual InhumanSpeech SpeechType { get; set; } = null;
+        public virtual InhumanSpeech SpeechType { get; set; }
 
         [CommandProperty(AccessLevel.GameMaster)]
-        public virtual OppositionGroup OppositionGroup { get; set; } = null;
+        public virtual OppositionGroup OppositionGroup { get; set; }
 
         [CommandProperty(AccessLevel.GameMaster)]
-        public virtual TimeSpan ReacquireDelay { get; set; } = TimeSpan.FromSeconds(10.0);
+        public virtual TimeSpan ReacquireDelay { get; set; }
```

Comment:
Curious, would this default somewhere else or cause any issues wihtout being defined?

Reply:
Good catch will fix, shouldn't have removed that default. This would cause them to instantly re-aggro after losing a target.